ebos/eclproblem.hh

```diff
@@ -1193,8 +1193,10 @@ public:
                 values.setThermalFlow(context, spaceIdx, timeIdx, boundaryFluidState(globalDofIdx, indexInInside));
             else if (type == BCType::FREE || type == BCType::DIRICHLET)
                 values.setFreeFlow(context, spaceIdx, timeIdx, boundaryFluidState(globalDofIdx, indexInInside));
-            else
+            else if (type == BCType::RATE)
                 values.setMassRate(massrate, pvtRegionIdx);
+            else
```

Comment:
won't this overwrite the thermal condition from the if above this nest?

Reply:
I dont think we use enableThermalFluxBoundaries much but youre right. I will fix. Thanks for noticing.